src/Bundle/CoreBundle/Controller/GraphQLApiController.php

```diff
@@ -34,12 +35,13 @@ public function indexAction(Organization $organization, Domain $domain, Request
         $schema = new Schema(
             [
                 'query' => $schemaTypeManager->getSchemaType('Query'),
-                'mutation' => $schemaTypeManager->getSchemaType('Mutation'),
+                'mutation' => ($domain->hasContentTypes()) ? $schemaTypeManager->getSchemaType('Mutation'):NULL,
```

Comment:
you should also check for ContentTypes. Maybe give the domain a method hasDefinedTypes() or hasContentOrSettingTypes()